src/hooks/protections/receive_net_message.cpp

```diff
@@ -23,6 +23,15 @@ namespace big
 			return false;
 	}
 
+	void gamer_handle_deserialize(rage::rlGamerHandle& hnd, rage::datBitBuffer& buf)
+	{
+		if ((hnd.m_platform = buf.Read<uint32_t>(8)) != 3)
```

Comment:
ideally we'd want a variable for this `3`

Reply:
Why would we want a variable for the 3? It will never change.

Reply:
https://en.wikipedia.org/wiki/Magic_number_(programming)